br/pkg/task/stream.go

```diff
@@ -676,9 +676,19 @@ func RunStreamRestore(
 		return errors.Trace(err)
 	}
 
-	// TODO split put and delete files
 	// perform restore kv files
-	return client.RestoreKVFiles(ctx, rewriteRules, datas)
+	if err := client.RestoreKVFiles(ctx, rewriteRules, datas); err != nil {
+		return errors.Trace(err)
+	}
+
+	for _, table := range fullBackupTables {
```

Comment:
maybe fullBackupTables's table info is outdate?

Reply:
I guess that was out of our scope 🤔. If the table list is stale, `RestoreKVFiles` would skip those tables too. For handling incremental tables, we should apply the meta key change at the `fullBackupTables`(Which hasn't been implemented for now).